### PR TITLE
Submitting Cerified Connector artifects for Candidate Zip

### PR DESCRIPTION
**Pull request by CandidateZip**
Submitting Cerified Connector artifects for Candidate Zip

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### CODE REVIEW

File: certified-connectors/CandidateZip/readme.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+## CandidateZip Connector
+	CandidateZip is a Global Leader of Simplified Resume/Job Data Extraction into ATS/CRM for Recruiters.
+Typically, we help recruiters who need to extract resumes or jobs into their ATS/CRM quickly. They can easily set up this automated extraction process in three easy steps. We help them save time on data entry so that they can do what they love to (close more jobs).  
+
+
+## Pre-requisites
+You will need the following to proceed:
+* Active Microsoft Flow login
```
Comment (dougelliott) [inline certified-connectors/CandidateZip/readme.md]: Please change to "Active Microsoft Power Automate, Power Apps, or Logic Apps account"

**Comment by dougelliott** [comment]
@CandidateZip The CLA will need to be signed before we are able to proceed with open sourcing. Please see the comment by @msftclas above for steps on completing that. Thanks!

File: certified-connectors/CandidateZip/apiProperties.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  "properties": {
+    "capabilities": [],
+    "connectionParameters": {
+      "token": {
+        "oAuthSettings": {
+          "clientId": "<client-id-please-replace>",
+          "customParameters": {
+            "authorizationUrl": {
+              "value": "https://<replace with candidatezip authorization url>"
+            },            ,
```
Comment (sriyen-msft) [inline certified-connectors/CandidateZip/apiProperties.json]: @CandidateZip Please remove the extra comma added after the value of authorizationUrl

Comment (CandidateZip) [reply certified-connectors/CandidateZip/apiProperties.json]: I submitted the changes.

**Comment by CandidateZip** [comment]
@dougelliott i made the changes as requested

**Comment by CandidateZip** [comment]
Please proceed further. I also signed CLA

**Pull request merged**